microshift_updating/microshift-update-options.adoc

```diff
@@ -63,7 +79,11 @@ You can update {op-system-ostree} or {op-system} without updating {microshift-sh
 
 [id="microshift-update-options-simultaneous-microshift-rhel-updates_{context}"]
 == Simultaneous {microshift-short} and operating system updates
-You can update {op-system-ostree} or {op-system} and update {microshift-short} at the same time, on the condition that the versions are compatible. Check for compatibility before beginning an update. First use the {op-system-ostree} and {op-system} documentation specific to your update path to plan and update the operating system. Then use the {microshift-short} update type specific to your update path.
+You can update {op-system-ostree} or {op-system} and update {microshift-short} at the same time, on the condition that the versions are compatible.
+
+* Check for compatibility before beginning an update.
+* First use the {op-system-ostree} and {op-system} documentation specific to your update path to plan and update the operating system.
+* Then use the {microshift-short} update type specific to your update path.
```

Comment:
There could be another bullet below this about enabling the correct MicroShift repository before starting the simultaneous upgrade. If you do not enable, you'll end up with with an upgraded OS, but not MicroShift. Yesterday, I went from a RHEL 9.2 system running MicroShift 4.15 to a RHEL 9.4 system running MicroShift 4.15 (unsupported).